chapter/8/big-data.md

```diff
@@ -1,11 +1,229 @@
 ---
 layout: page
 title:  "Large Scale Parallel Data Processing"
-by: "Joe Schmoe and Mary Jane"
+by: "Jingjing and Abhilash"
 ---
+## Introduction
+`JJ: Placeholder for introduction` The booming Internet has generated big data...
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. {% cite Uniqueness --file big-data %}
 
-## References
+This chapter is organized in <label for="note1" class="margin-toggle sidenote-number"></label><input type="checkbox" id="note1" class="margin-toggle"/><span class="sidenote">JJ: need to fill in more stuff</span>
+
+- **Data paralleling**:
```

Comment:
I am sorry professor, the section "Introduction" shouldn't have been there. It missed my eyes. Its speaking about the old draft and as we can see the new draft could be like below.
 - Map Reduce programming model
 - Problems with Iterative map reduce and new frameworks to handle such iterative algorithms
 - Spark & DyradLinq - A new functional programming way of doing Map reduce
 - Declarative interfaces - easy programming and opportunities for optimizations
- A special model - SparkSQL which enables the intermixing of both relational and procedural
 - Frameworks with optimizers
 - A brief introduction to a parallel world of large scale graph processing -> their limitations -> and how GraphX overcame it by leveraging Spark to address graph processing algorithms.

I am really sorry for this. I am working on this with Jingjing. We will meet you this monday to revisit the flow.